test/e2e/mysql_kafka.yml

```diff
@@ -4,7 +4,6 @@ source:
   name: mysql
   config:
     connection_url: root:admin@tcp(127.0.0.1:3306)/
-    identifier: test_db
```

Comment:
how was this used earlier?

Reply:
same as mentioned above, was earlier used for URN generation